server/mdm/maintainedapps/testing_utils.go

```diff
@@ -0,0 +1,74 @@
+package maintainedapps
+
+import (
+	"context"
+	"encoding/json"
+	"net/http"
+	"net/http/httptest"
+	"os"
+	"path"
+	"path/filepath"
+	"runtime"
+	"testing"
+
+	"github.com/fleetdm/fleet/v4/server/fleet"
+	"github.com/go-kit/log"
+	"github.com/stretchr/testify/require"
+)
+
+// IngestMaintainedApps ingests the maintained apps from the testdata
+// directory, to fill the library of maintained apps with valid data for tests.
+// It returns the expected results of the ingestion as a slice of
+// fleet.MaintainedApps with only a few fields filled - the result of
+// unmarshaling the testdata/expected_apps.json file.
+func IngestMaintainedApps(t *testing.T, ds fleet.Datastore) []*fleet.MaintainedApp {
```

Comment:
Heads-up, those should generally be used for tests when the maintained apps are needed, will generate a cohesive, valid set of apps.